recipes/igraph/build.sh

```diff
@@ -0,0 +1,6 @@
+export CFLAGS=-I${PREFIX}/include
+./configure \
+	--prefix=$PREFIX 
+make -j $CPU_COUNT
+#make check
```

Comment:
I uncommented it. It didn't use to pass when I generated this recipe, it was a *long* time ago.